x-pack/plugins/security_solution/cypress/integration/exceptions/rule_details_flow/add_exception.spec.ts

```diff
@@ -0,0 +1,221 @@
+
+  before(() => {
+    esArchiverResetKibana();
+    esArchiverLoad('exceptions');
+    login();
+  });
+
+  after(() => {
+    esArchiverUnload('exceptions');
+  });
+
+  describe('rule with existing exceptions', () => {
+    const exceptionList = getExceptionList();
+    beforeEach(() => {
+      deleteAlertsAndRules();
+      deleteExceptionList(exceptionList.list_id, exceptionList.namespace_type);
+      // create rule with exceptions
+      createExceptionList(exceptionList, exceptionList.list_id).then((response) => {
+        createCustomRule(
+          {
+            ...getNewRule(),
+            customQuery: 'agent.name:*',
+            dataSource: { index: ['exceptions*'], type: 'indexPatterns' },
+            exceptionLists: [
+              {
+                id: response.body.id,
+                list_id: exceptionList.list_id,
+                type: exceptionList.type,
+                namespace_type: exceptionList.namespace_type,
+              },
+            ],
+          },
+          '2'
+        );
+        createExceptionListItem(exceptionList.list_id, {
+          list_id: exceptionList.list_id,
+          item_id: 'simple_list_item',
+          tags: [],
+          type: 'simple',
+          description: 'Test exception item',
+          name: 'Sample Exception List Item',
+          namespace_type: 'single',
+          entries: [
+            {
+              field: 'user.name',
+              operator: 'included',
+              type: 'match_any',
+              value: ['bar'],
+            },
+          ],
+        });
+        createExceptionListItem(exceptionList.list_id, {
+          list_id: exceptionList.list_id,
+          item_id: 'simple_list_item_2',
+          tags: [],
+          type: 'simple',
+          description: 'Test exception item 2',
+          name: 'Sample Exception List Item 2',
+          namespace_type: 'single',
+          entries: [
+            {
+              field: 'unique_value.test',
+              operator: 'included',
+              type: 'match_any',
+              value: ['foo'],
+            },
+          ],
+        });
+      });
+
+      visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+      goToRuleDetails();
+      goToExceptionsTab();
+    });
+
+    it('Creates an exception item', () => {
+      // displays existing exception items
+      cy.get(NO_EXCEPTIONS_EXIST_PROMPT).should('not.exist');
+      cy.get(EXCEPTION_ITEM_VIEWER_CONTAINER).should('have.length', 2);
+
+      // clicks prompt button to add a new exception item
+      addExceptionFromRuleDetails(getException());
+
+      // new exception item displays
+      cy.get(EXCEPTION_ITEM_VIEWER_CONTAINER).should('have.length', 3);
+    });
+
+    // Trying to figure out with EUI why the search won't trigger
+    it.skip('Can search for items', () => {
+      // displays existing exception items
+      cy.get(NO_EXCEPTIONS_EXIST_PROMPT).should('not.exist');
```

Comment:
Same here!

Reply:
Thank you thank you for your feedback. I've opened a follow up PR addressing it [here](https://github.com/elastic/kibana/pull/140448).